### ISSUE TITLE
IndexOutOfBoundsException

### ISSUE THREAD

**Issue by amohoste** [RMLio/rmlmapper-java#3]
When I run the rmlmapper-java I get the following exception:

```
Exception in thread "main" 
java.lang.IndexOutOfBoundsException: Index: 0, Size: 0

	at java.util.ArrayList.rangeCheck(ArrayList.java:653)
	at java.util.ArrayList.get(ArrayList.java:429)

	at be.ugent.rml.Executor.generatePredicateObjectsForSubject(Executor.java:80)

	at be.ugent.rml.Executor.execute(Executor.java:60)
	at be.ugent.rml.cli.Main.main(Main.java:96)
```

When I run RML-Mapper with the same file I don't get an exception and everything works fine, so I think this must be a code bug.

RML file used:
[file.txt](https://github.com/RMLio/rmlmapper-java/files/2265617/file.txt)

**Comment by pheyvaer** [comment]
Is it possible to provide the input file too?

**Comment by amohoste** [comment]
Sure, here is a zip with the RML file in file.txt and the other files included. I have changed changed the sources in the rml file so the included files are used.

[Archive.zip](https://github.com/RMLio/rmlmapper-java/files/2266476/Archive.zip)

**Comment by pheyvaer** [comment]
Fixed with version 0.1.6
